<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RescanCollectionDialog</class>
 <widget class="QDialog" name="RescanCollectionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rescanning collection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageScanFiles">
      <property name="styleSheet">
       <string notr="true">QTextEdit {
border-top: 1px solid rgb(210,210,210);
border-bottom: 1px solid rgb(210,210,210);
border-left: 0px;
border-right: 0px;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="labelStatus">
           <property name="text">
            <string>Scanning collection...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageConfirmSerienjunkies">
      <property name="styleSheet">
       <string notr="true">QLabel#labelSeriesTitle2 {
font-size: 20px;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>-1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>You have added a new TV show to your collection. Enter its Serienjunkies URL below!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSeriesTitle2">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>URL:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxSerienjunkiesUrl">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Searching show...</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageConfirmTvdb">
      <property name="styleSheet">
       <string notr="true">QLabel#labelSeriesTitle {
font-size: 20px;
}

QFrame#frameDetails {
border-top: 1px solid rgb(210,210,210);
border-bottom: 1px solid rgb(210,210,210);
border-left: 0px;
border-right: 0px;
}
QWidget#seriesWidgetBackground {
background-color: qlineargradient(x1:0 y1:0, x2:0 y2:1, stop:0 rgb(225,225,225) stop:1 rgb(250,250,250));
background-image: url(:/general/noisy-texture-200x200);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>You have added new TV shows to your collection. Find their correct TVDB entry!</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelSeriesTitle">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEditSearch"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonSearch">
             <property name="text">
              <string>Search</string>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frameDetails">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="opaqueResize">
             <bool>false</bool>
            </property>
            <property name="handleWidth">
             <number>1</number>
            </property>
            <widget class="QTreeView" name="treeView">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QWidget" name="seriesWidgetBackground" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="SeriesWidget" name="seriesWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonIgnoreFolder">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ignore folder</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelProgress">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSkip">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonContinue">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeriesWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/serieswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
